### ISSUE TITLE
Add HVAC screen logic

### ISSUE THREAD

**Issue by ryanjduffy** [enactjs/agate-apps#32]
Everything is “local” state right now but the app should behave differently when you click “Auto” which would likely disable or set the fan speed and disable or set the fan direction buttons.

This could be superseded by connecting up to system services which would likely "own" the logic for these settings so we might want to wait on this.

**Comment by ryanjduffy** [comment]
➡️ PLAT-68620
